tests/20-packet-parsing/packet-injector.sh

```diff
@@ -21,8 +21,19 @@ make -C $CODE_DIR TARGET=native > make.log 2> make.err
 
 for i in $PACKET_DIR/*
 do
-  export TEST_FILE=$i
-  echo Injecting file $TEST_FILE
+  if [ -d "$i" ]; then
+    export TEST_FILES=0
+    for f in $i/*
+    do
+      export TEST_FILE_$(printf "%02d" $TEST_FILES)=$f
```

Comment:
Can't the existing `TEST_FILE` be renamed to `TEST_FILES` and generalized into a list of files separated by some suitable character instead of this dynamic environment variable creation? (That should give 100% coverage of the affected parts of the C code from a single test.)

Reply:
Yes, a single environment variable would be cleaner. Another option is to make it possible to pass files as command line arguments using the contiki_argc and contiki_argv variables.

Reply:
I switched to `contiki_argc`/`argv` for test files.